celesta-maven-plugin/src/main/java/ru/curs/celesta/plugin/maven/CursorGenerator.java

```diff
@@ -219,7 +258,10 @@ private static String calcClassName(GrainElement ge, String sourceFileNamePrefix
                 .map(
                         c -> {
                             TypeSpec.Builder builder = TypeSpec.classBuilder(StringUtils.capitalize(c.getName()))
-                                    .addModifiers(Modifier.PUBLIC, Modifier.STATIC, Modifier.FINAL);
+                                    .addModifiers(Modifier.PUBLIC, Modifier.STATIC, Modifier.FINAL)
+                                    .addAnnotation(AnnotationSpec.builder(Generated.class)
+                                            .addMember("value", "$S", "Generated by ru.curs.celesta.plugin.maven.CursorGenerator")
```

Comment:
1. Instead of hard-coded fully qualified class name, it's better to use `getClass().getName()`.
2. "Generated by" phrase may be omitted, it makes the code noisy and does not add any value. From `@Generated("ru.curs....")` we can plainly see that the code was generated and what exactly generated it.
3. `@Generated` annotation also has [`date` attribute](https://docs.oracle.com/javase/8/docs/api/javax/annotation/Generated.html). It might be handy to have it also included. Of course, something should be devised for the tests to pass.

Reply:
Addressed